tests/integration/targets/at/tasks/main.yml

```diff
@@ -54,9 +54,55 @@
 - debug: var=at_test0
 - name: validate results
   assert:
-      that:
-          - 'at_test0.changed is defined'
-          - 'at_test0.count is defined'
-          - 'at_test0.script_file is defined'
-          - 'at_test0.state is defined'
-          - 'at_test0.units is defined'
+    that:
+      - 'at_test0.changed is defined'
+      - 'at_test0.count is defined'
+      - 'at_test0.script_file is defined'
+      - 'at_test0.state is defined'
+      - 'at_test0.units is defined'
+
+- name: add a at useless command
+  at:
+    command: /bin/logger 'AT task ran from Ansible'
+    count: 1
+    units: minutes
+    unique: yes
+  register: at_add
+
+- debug: var=at_add
+
+- name: pause, giving the time the at runs once
+  pause:
+    minutes: 1
+
+- name: add a at useless command
+  at:
+    command: /bin/logger 'AT task ran from Ansible'
+    count: 1
+    units: minutes
+    unique: true
+    state: absent
+  register: at_removal
+  ignore_errors: true
+
+- debug: var=at_removal
+
+- name: create an at command with chdir with valid value
+  at:
+    command: /bin/logger 'AT task ran from Ansible with chdir'
+    count: 1
+    units: minutes
+    chdir: /tmp
+  register: at_chdir_valid
+- debug: var=at_chdir_valid
+
+- name: create an at command with chdir with invalid value
+  at:
+    command: /bin/logger 'AT task ran from Ansible with chdir'
+    count: 1
+    units: minutes
+    chdir: /invalid
+  register: at_chdir_invalid
+  ignore_errors: true
+
+- debug: var=at_chdir_invalid
```

Comment:
There are 2 `ignore_errors: true` but nothing to manage/handle the how and why the failure, and even to ensure a failure happened when expected. Typically, such tasks should end with an `assert` task to check previously registered results, as above.